mne/viz/misc.py

```diff
@@ -335,8 +335,9 @@ def _plot_mri_contours(mri_fname, surfaces, src, orientation='coronal',
                               zorder=1)
 
         for sources in src_points:
-            in_slice = np.logical_and(sources[:, z] > sl - 0.5,
-                                      sources[:, z] < sl + 0.5)
+            slice_size = np.diff(np.unique(np.round(sources[:, z]))).min()
```

Comment:
Probably better to use `img.header.get_zooms()[2]`?

Reply:
maybe. What does `img` refers to in this case?

Reply:
Apparently here we call in `nim`

https://github.com/mne-tools/mne-python/blob/maint/0.19/mne/viz/misc.py#L306

Reply:
... basically the idea is that you should use the `zooms` from the MRI to know how big each voxel is. And you should use the zoom index (0, 1, or 2) based on which of the three views is being shown

Reply:
ok that doesnt work. get_zoom() returns 1, 1, 1 and get s it wrong

Reply:
Unclear to me. Do you want to take the lead? `tutorial/plot_forward.py` faces this issue so it's easy to have a look

Reply:
yeah I can look